sdk/cognitivelanguage/perf-tests/ai-language-text/README.md

```diff
@@ -0,0 +1,11 @@
+# Performance Testing for Azure Text Analysis
+
+## Instructions
+
+1. Build the ai-language-text perf tests package `rush build -t perf-ai-language-text`.
+2. Copy the `sample.env` file and name it as `.env`.
+3. Create a cognitive services account and populate the `.env` file with the relevant credentials.
```

Comment:
maybe we should say "Language service" account here?

Reply:
Good point, done.